contrib/transaction_profiling_analyzer/transaction_profiling_analyzer.py

```diff
@@ -50,9 +50,10 @@
 PROTOCOL_VERSION_6_3 = 0x0FDB00B063010001
 PROTOCOL_VERSION_7_0 = 0x0FDB00B070010001
 PROTOCOL_VERSION_7_1 = 0x0FDB00B071010001
+PROTOCOL_VERSION_7_2 = 0x0FDB00B072010001
```

Comment:
We unset the last bit of the protocol version recently, so this should be either `0x0FDB00B072010000` or `0x0FDB00B072000000`, depending on whether we decide to unset the dev digit as described below.

Reply:
fixed same as on `ProtocolVersion.h`